mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,20 @@
 
 */
 
-function mood() {
-  let isHappy = true;
+function mood(e) {
```

Comment:
I think "e" is too short for any abbreviation, and it is always recommended to use more describing names